src/main/kotlin/racingcar/entity/Car.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.entity
+
+class Car(private var name: Name) {
```

Comment:
name은 변경이 필요없어 보여요! 
항상 불변으로 선언해주시면 좋습니다!
```suggestion
class Car(private val name: Name) {
```